wiki/en/Include-Backing-Up.md

```diff
@@ -1,4 +1,4 @@
-<!-- NOTE: This must apply to both client and server, and all operating systems -->
+<!-- NOTE: This must apply to both client and Server, and all operating systems -->
```

Comment:
Hm.

Reply:
You've been inconsistent over "client" capitalisation.

Reply:
So if "Server" is capitalised, we should have "Client" as well you mean? I guess that makes sense. Right now it's always "client" and "Server" (unless "server" refers to something that isn't Jamulus).

Reply:
Yeah, we've "defined" the terms "Client", "Server" and "Directory", so they should be treated equally when used per definition.